File: api/v2010/outgoing_caller_id/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

outgoing_caller_id = @client
  .outgoing_caller_ids('PNe536d32a3c49700934481addd5ce1659')
  .update(friendly_name: 'My Second Line')

puts outgoing_caller_id.friendly_name

File: api/v2010/outgoing_caller_id/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var outgoingCallerId = OutgoingCallerIdResource.Update(
            friendlyName: "My Second Line",
            pathSid: "PNe536d32a3c49700934481addd5ce1659"
        );

        Console.WriteLine(outgoingCallerId.FriendlyName);
    }
}
